<launch>
  <param name="use_sim_time" value="false"/>
  <node pkg="tf" type="static_transform_publisher" name="base_to_velodyne" args="0.0 0.0 0.0 0.0 0.0 0.0 base_link velodyne 100"/>
  <node name="lidar_slam_3d" pkg="lidar_slam_3d" type="lidar_slam_3d" output="screen">
    <param name="base_frame"                 value="base_link"/>
    <param name="map_frame"                  value="map"/>
    <param name="publish_freq"               value="0.2"/>
    <param name="min_scan_distance"          value="5"/>
    <param name="point_cloud_topic"          value="velodyne_points"/>
    <param name="enable_floor_filter"        value="false"/>
  </node>
</launch>
